OpenRA.Mods.Common/Widgets/RadarWidget.cs

```diff
@@ -243,17 +243,20 @@ public override bool HandleMouseInput(MouseInput mi)
 
 			var cell = MinimapPixelToCell(mi.Location);
 			var pos = world.Map.CenterOfCell(cell);
-			if ((mi.Event == MouseInputEvent.Down || mi.Event == MouseInputEvent.Move) && mi.Button == MouseButton.Left)
+			if ((mi.Event == MouseInputEvent.Down || mi.Event == MouseInputEvent.Move)
+				&& mi.Button == Game.Settings.Game.MouseButtonPreference.Cancel)
+			{
 				worldRenderer.Viewport.Center(pos);
+			}
```

Comment:
I think in this case the braces really help with readability as the indention alone doesn't really make things clear.